src/app/components/attestation/attestation-create-exception.jsx

```diff
@@ -17,6 +17,7 @@ import { developer as developerPropType } from 'shared/prop-types';
 const useStyles = makeStyles({
   buttonMargin: {
     marginTop: '8px',
+    marginLeft: '4px',
```

Comment:
Will this move both buttons to the right? Are they currently "left-aligned"? Will this change that?

Reply:
This will create spacing in between the buttons. On our other dialogs pop ups we have the buttons left aligned.

Please check the confirmation dialog for an example.